core/domain/stats_domain.py

```diff
@@ -189,6 +171,9 @@ def __init__(
         self.total_hit_count_v2 = total_hit_count_v2
         self.first_hit_count_v1 = first_hit_count_v1
         self.first_hit_count_v2 = first_hit_count_v2
+        # There is no version 1 for the solution viewed count because
+        # solutions were introduced recently and there are no existing event
```

Comment:
"recently" won't make sense after a while. Maybe instead say: "because solution view analytics were only introduced in v2, and there are no existing event models in v1 that record 'solution-viewed' events".

Reply:
Done.